handbook/ce/team-culture.md

```diff
@@ -0,0 +1,181 @@
+
+#### How we get there
+As we lay out the framework for being able to make some of these pivots, we must be thoughtful of the impact to other parts of the company and we must ensure we are always doing right by our customers.
+
+CE guiding principles remain unchanged: we will continue to work across both pre- and post-sales activities and are laser focused on our customers being able to use Sourcegraph well.
+
+##### Strategic Focus
+A subset of the CE team will assume a strategic account focus; these individuals will not be removed from any existing opportunities that they are supporting. CE leadership is accountable for providing a plan (timing and activities) to Sales leadership with a recommendation to realign any impacted accounts from CE perspective only. No accounts with active opportunities, renewals, or expansion opportunities will be impacted or disrupted. This is expected to be complete and in place by the end of Q2.
+
+##### Project-based Focus
+CEs have already begun and delivered on several important internal project work (customer-facing demo recordings, use-case based demo scripts, customer-facing deployment guides). All CEs, based on their existing work load and customer needs, have opportunities to contribute and lead projects. This is why 1-2 week sprints are critical framing for project work (more below). CE leadership will compile from cross-functional teams an initial set of projects and define each project against the templated criteria below. CEs, based on their availability, will be assigned to lead individual projects and work with the applicable stakeholders to get inputs, reviews / approvals, etc.
+
+As we continue CE hiring and onboarding in Q2 and beyond, and as we assess both the needs and impact of this project work for our teams and our customers, we will gauge whether there exists enough need to have CEs be exclusively dedicated to project work in the future.
+
+For CEs to be able to execute on these critically important areas will require availability and focus. As such, this exercise has brought to light the need to clarify and/or realign some specific areas to other parts of the company.
+
+#### Teams with whom we have aligned
+Product:
+* For NPS feedback, PMs should review and follow up. For our strategic accounts, Product will work with the AE and the CE to address any feedback and participate in weekly product roadmap reviews or monthly feedback theme reviews.
```

Comment:
Does "follow-up" here entail also emailing the customer? (Right now I do some amount of "hi, customer, you said we can't do this and we actually can, here's the doc.")

Reply:
The intent is for product to reach out to customers. Of course can always work with AE / CE to determine best strategy.

Reply:
Fair enough! From my perspective: Product won't want to respond to everything (sometimes it's just a confused individual user) and that's fine, but I want to make sure the individual users in my accounts get the info they need from a success perspective, so I'd prefer to respond to them. But perhaps I'm missing something/am offbase here.